docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -108,76 +103,103 @@ deviceCommands:
+    $ git checkout main
+    ```
+    !!! note
+        Use **main** branch until **jakarta** is released.
 2. Generate the docker-compose.yml file (notice this includes mqtt-broker)
-```
-$ cd edgex-compose/compose-builder
-$ make gen ds-mqtt mqtt-broker no-secty ui
-```
-Check the generated file
-```
-$ ls | grep 'docker-compose.yml'
-docker-compose.yml
-```
+    ```
+    $ cd edgex-compose/compose-builder
+    $ make gen ds-mqtt mqtt-broker no-secty ui
+    ```
+3. Check the generated file
+    ```
+    $ ls | grep 'docker-compose.yml'
+    docker-compose.yml
+    ```
 
 ### Mount the custom-config
 
-Open the `docker-compose.yml` file and then add volumes path and environment as shown below:
+Open the `edgex-compose/compose-builder/docker-compose.yml` file and then add volumes path and environment as shown below:
 
 
 ```yaml
+ # docker-compose.yml
+
  device-mqtt:
     ...
     environment:
-      ...
       DEVICE_DEVICESDIR: /custom-config/devices
       DEVICE_PROFILESDIR: /custom-config/profiles
+      ...
     volumes:
+     - /path/to/custom-config:/custom-config
     ...
-    - /path/to/custom-config:/custom-config
 ```
 
 !!! note
     Replace the `/path/to/custom-config` in the example with the correct path
 
+## Enabling Multi-Level Topics
+
+To use the optional setting for MQTT device services with multi-level
+topics, make the following changes in the device service configuration files:
+
+1. There are two ways to set the environment variables for multi-level topics. 
+
+   1. If the code is built with compose builder, modify the docker-compose.yml file in edgex-compose/compose-builder:
+
+      ```yaml
+      # docker-compose.yml
+      
+      device-mqtt:
+        ... 
+        environment:
+          MQTTBROKERINFO_INCOMINGTOPIC: "incoming/data/#"
+          MQTTBROKERINFO_RESPONSETOPIC: "command/response/#"
+          MQTTBROKERINFO_USETOPICLEVELS: "true"
+          ...
+      ```
+
+   2. Otherwise if the device service is built locally, modify these lines in `configuration.toml`:
+
+      ``` toml
+      # Comment out/remove when using multi-level topics
+      #IncomingTopic = "DataTopic"
+      #ResponseTopic = "ResponseTopic"
+      #UseTopicLevels = false
+      
+      # Uncomment to use multi-level topics
+      IncomingTopic = "incoming/data/#"
+      ResponseTopic = "command/response/#"
+      UseTopicLevels = true
+      ```
+
+      !!! note
+          If you have previously run Device MQTT locally, you will need to remove the services configuration from Consul. This can be done with:
+      `curl --request DELETE http://localhost:8500/v1/kv/edgex/devices/2.0/device-mqtt?recurse=true`
+
+2. In  `my.custom.device.config.toml`:
+
+   ``` toml
+   [DeviceList.Protocols]
+    [DeviceList.Protocols.mqtt]
+       # Comment out/remove below to use multi-level topics
+       # CommandTopic = "CommandTopic"
+       # Uncomment below to use multi-level topics
+       CommandTopic = "command/my-custom-device"
+   ```
+
```

Comment:
Do we need a similar note for deleting the device if it was previously loaded? I'll leave the how for you to determine via the [API reference](https://docs.edgexfoundry.org/2.0/api/core/Ch-APICoreMetadata/) :-)